nixos/release-combined.nix

```diff
@@ -66,12 +66,11 @@ in rec {
         (all nixos.tests.containers-ip)
         nixos.tests.chromium.x86_64-linux or []
         (all nixos.tests.firefox)
-        (all nixos.tests.firefox-esr)
+        #(all nixos.tests.firefox-esr)
         (all nixos.tests.firewall)
         (all nixos.tests.fontconfig-default-fonts)
         (all nixos.tests.gnome3-xorg)
         (all nixos.tests.gnome3)
-        (all nixos.tests.pantheon)
```

Comment:
:eyes: I could suggest better tests to remove than something I worked months on to ensure is working, and an environment that is actually rather popular.

Reply:
Given the complexity of the pantheon ecosystem and the fact that it is one test, I am +1 on putting pantheon back.

Reply:
there's also gnome's installed tests which are highly helpful for us that are commented https://github.com/NixOS/nixpkgs/blob/master/nixos/tests/installed-tests/default.nix. But I do understand that's about 17 test cases.